<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookWidget</class>
 <widget class="QWidget" name="AddBookWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加新书</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>采购入库日期：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="m_publishDateEdit_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>88</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>书号-留空则自动生成：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_titleLineEdit_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>132</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>42</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>书名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_titleLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>42</y>
     <width>132</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>80</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_authorLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>132</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>200</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>出版日期：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="m_publishDateEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>88</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_publishLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>132</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>出版社：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_priceSpinBox">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>160</y>
     <width>53</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>价格：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_addButton">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>333</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确认添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_returnButton">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>333</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
